### PR TITLE
Add psu hardware revision to psushow table

### PR DESCRIPTION
**Pull request by robocoder99**
#### What I did

Added a field for hardware revision to the psushow output table. 

#### How I did it

Defined new psu status field sourced from "revision" field in STATE_DB added in https://github.com/robocoder99/sonic-platform-daemons/pull/1 and adds it to the table output with the header "HW Rev"

All relevant unit tests were also updated.

#### How to verify it

Execute `show platform psustatus` and verify that "HW Rev" column is present and appropriately populated depending on the platform. 

#### Previous command output (if the output of a command-line utility has changed)

```
PSU    Model    Serial      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  -------  --------  -------------  -------------  -----------  --------  -----
PSU 1  N/A      N/A               12.08           3.62        43.75  OK        green
```

#### New command output (if the output of a command-line utility has changed)

```
PSU    Model          Serial        HW Rev      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  -------------  ------------  --------  -------------  -------------  -----------  --------  -----
PSU 1  MTEF-PSF-AC-A  MT1621X15246  A3                11.97           4.56        54.56  OK        green
```

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@robocoder99 can you please check the dependency in the comment? we need to have the public PR.

File: tests/mock_tables/state_db.json [hunk 1]
```diff
@@ -141,6 +141,7 @@
         "status": "true",
         "model": "0J6J4K",
         "serial": "CN-0J6J4K-17972-5AF-0086-A00",
+	"revision": "1",
```
Comment (liat-grozovik) [inline tests/mock_tables/state_db.json:144]: please fix indentation

**Review by liat-grozovik — CHANGES_REQUESTED** [review]
please review the command reference guide and see if cli command is there and update the output based on it.
@jleveque do we have other documentation of the platform commands?

**Comment by robocoder99** [comment]
No dependency, this PR will safely handle if the data is not present in STATE_DB (it will be shown as blank if the PSU is present and it will show as N/A if it is not)

**Comment by robocoder99** [comment]
There is an example in the command reference for `show platform psustatus` that is currently shown as:

```admin@sonic:~$ show platform psustatus
PSU    Status
-----  --------
PSU 1  OK
PSU 2  OK```

I can add an update to this PR.

**Review by jleveque — CHANGES_REQUESTED** [review]
Agree with Liat - please fix indentation.

**Review by jleveque — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**